README.md

```diff
@@ -33,10 +32,14 @@ Or via pip:
 $ pip install genomepy
 ```
 
-If you install via pip, you will have to install some dependencies, only if 
-you want to use the annotation download feature. 
-You will have to install the following
-utilities and make sure they are in your PATH:
+Installation via pip requires `pypandoc` installed. Additionally, 
```

Comment:
Yeah, this kinda sucks.The problem is that pypandoc is necessary for only one thing: making sure the description on pypi is correct. Pypi used to only support rst README format. However, it should now support markdown (md) as well: https://pypi.org/project/markdown-description-example/ 
When I tried this previously it didn't work reliably, but we can try it again. (Worst case scenario: the README doesn't look nice on pypi).

Reply:
Since I cannot upload this repo to testPypi, I cannot check it. I'll leave this for you :)

If you decide to remove pypandoc, references to it are in the `README`, `setup.py` and the `enviroment.yml`.

Since I removed setup.cfg, the README might already be gone. Needs to be tested on testPypi too.